.travis.yml

```diff
@@ -1,23 +1,17 @@
-sudo: required
 dist: trusty
+sudo: false
 language: python
 python:
-- '2.7'
+  - "2.7.13"
+  - "3.6.1"
```

Comment:
Do these versions need to be so specific as 3.6.x , 2.7.x instead of 3.6, 2.7?

Reply:
Done.